manifests/vizier/core-rest/rbac.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Since core-rest does not need to manage k8s resources, the rbac looks unnecessary.

Reply:
good point, thanks for catching this!